CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
```

Comment:
[nitpick]: pls get rid of the `<br>`eaks in favor of normal newlines or enumerations/lists :)